cis194/week6/tien/src/Fibonacci.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE FlexibleInstances #-}
+module Fibonacci where
+
+fib :: Integer -> Integer
+fib 0 = 0
+fib 1 = 1
+fib n = fib (n-1) + fib (n-2)
+
+-- exercise 1 --
+fibs1::[Integer]
+fibs1 = map fib [0..]
+
+-- exercise 2 --
+fibs2::[Integer]
+fibs2 = 0: 1: next fibs2 where
+  next (x:y:xs) = (x+y) : next(y:xs)
```

Comment:
I remember @shineli1984 mentioned a while ago that we tend to put a space before and after the symbols in Haskell.

```suggestion
  next (x : y : xs) = (x + y) : next(y : xs)
```

Reply:
My apology....should have run the formatter before commits.